far/stddlg.hpp

```diff
@@ -65,16 +65,30 @@ struct SearchReplaceDlgProps
 struct SearchReplaceDlgParams
 {
 	string SearchStr;
-	bytes SearchBytes;
-	string ReplaceStr;
+	std::optional<bytes> SearchBytes;
+	std::optional<string> ReplaceStr;
 	std::optional<bool> Hex;
 	std::optional<bool> CaseSensitive;
 	std::optional<bool> WholeWords;
-	std::optional<bool> Regexp;
+	std::optional<bool> Regex;
 	std::optional<bool> Fuzzy;
 	std::optional<bool> PreserveStyle;
+
+	enum class SharedGroup { common, help, _count };
```

Comment:
Technically `_count` is not a reserved identifier, because it is not in the global namespace.

However, names like that give an impression that we are doing something fishy, like accessing some platform-specific bits, which is not the case. Maybe just `count`?

Reply:
Done!